src/tests/ftest/erasurecode/restart.py

```diff
@@ -31,43 +31,81 @@ def execution(self, agg_check=None):
                        after.Default is None so not to check aggregation, but wait for 20 seconds
                        and restart the servers.
         """
-        # Write all EC object data to NVMe
+        # 1.
+        aggr_threshold = self.params.get("aggregation_threshold", '/run/ior/*', default=300)
+        self.log_step("Create pool")
+
+        # 2.
+        self.log_step("Disable aggregation")
+        self.pool.set_property("reclaim", "disabled")
+        # Get initial total free space (scm+nvme)
+        initial_free_space = self.pool.get_total_free_space(refresh=True)
+        self.log.info("initial pool free space: %s", initial_free_space)
+
+        # 3.
+        self.log_step("Run IOR write all EC object data to container")
         self.ior_write_dataset(operation="Auto_Write", percent=self.percent)
```

Comment:
Use run_ior here instead - this should be a function as we'll call this twice:
```suggestion
        for container in containers:
            ior_kwargs['container'] = container
            try:
                result = run_ior(**ior_kwargs)
            except CommandFailure as error:
                self.fail('ior failed')
```